src/units.jl

```diff
@@ -21,9 +21,9 @@ end
 function _register_unit(name::Symbol, value)
     s = string(name)
     return quote
-        const $name = $value
```

Comment:
I think the symbols should still be constants though?

Reply:
Otherwise `using DynamicQuantities.Units: m`  – the `m` would be type unstable as its not a constant.

Reply:
I reverted to original.
Instead, I'm registering units outside testset. See [here](https://github.com/SymbolicML/DynamicQuantities.jl/pull/107/files#diff-97287d1f47874f9eb568bc53640b753ce7f6cd5fd70a18180a4beb9dd8a7939bR1852-R1854).